### PR TITLE
Create cortx-docker integration.

### PR DESCRIPTION
**Pull request by wb-droid**
Create cortx-docker integration.

-----
[View rendered doc/integrations/cortx-docker/README.md](https://github.com/wb-droid/cortx/blob/integration/doc/integrations/cortx-docker/README.md)

### CODE REVIEW

**Review by mukul-seagate11 — APPROVED** [review]
looks good

File: doc/integrations/cortx-docker/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    aws s3 --endpoint "http://192.168.1.91" ls s3://testbucket/docker/registry/v2/repositories/
+
+Step 3: From Docker deployment machine, pull docker image from Docker Registry server. Then you can start to use the downloaded docker image as usual.
+
+    C:\tmp>docker pull 192.168.1.115:5000/hello                                                                             
+    Using default tag: latest                                                                                               
+    latest: Pulling from hello                                                                                              
+    2db29710123e: Pull complete                                                                                             
+    Digest: sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4
```
Comment (mukul-seagate11) [inline doc/integrations/cortx-docker/README.md:91]: @wb-droid, can you remove or hide the digest

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: doc/integrations/cortx-docker/README.md (hunk 1 above)
Comment (wb-droid) [reply doc/integrations/cortx-docker/README.md]: @mukul-seagate11, sure, it's removed.

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Pull request merged**